From OmniOS

This test tries to set a file date to 2108-12-30.
On illumos, the os.utime() call raises:
	OSError: [Errno 79] Value too large for defined data type

--- Python-3.12.1/Lib/test/test_zipfile/test_core.py.orig
+++ Python-3.12.1/Lib/test/test_zipfile/test_core.py
@@ -622,7 +622,7 @@
             self.skipTest(f'time.localtime({ts}) raises OverflowError')
         try:
             os.utime(TESTFN, (ts, ts))
-        except OverflowError:
+        except (OverflowError, OSerror):
             self.skipTest('Host fs cannot set timestamp to required value.')
 
         mtime_ns = os.stat(TESTFN).st_mtime_ns
